pkgs/by-name/ug/ugrep/package.nix

```
@@ -40,28 +62,68 @@ stdenv.mkDerivation (finalAttrs: {
     zstd
   ];
 
-  postFixup = ''
-    for i in ug+ ugrep+; do
-      wrapProgram "$out/bin/$i" --prefix PATH : "$out/bin"
-    done
-  '';
+  postFixup =
+    with lib;
+    ''
+      # Needed because `ug+` and `ugrep+` are
+      # just scripts that call `ug` or `ugrep`
+      # with certain arguments. They must be
+      # reachable.
+      for i in ug+ ugrep+; do
+        wrapProgram "$out/bin/$i" --prefix PATH : "${
+          makeBinPath (
+            [ "$out" ]
+            ++ (optionals wrapWithFilterUtils [
+              poppler-utils
+              antiword
+              pandoc
+              exiftool
+            ])
+          )
+        }"
+      done
+    ''
+    + optionalString (createGrepReplacementLinks) ''
```

```suggestion
    + optionalString createGrepReplacementLinks ''
```

See the formatting failure.

---

Fixed.